src/ui/server/helpers/summary-lists/your-business/index.ts

```diff
@@ -13,32 +13,42 @@ import { ApplicationCompany, ApplicationBusiness, SummaryListItemData } from '..
  * @param {Number} referenceNumber
  * @returns {Object} All your business values in an object structure for GOVUK summary list structure
  */
-const generateFields = (answersCompany: ApplicationCompany, answersBusiness: ApplicationBusiness, referenceNumber: number, checkAndChange: boolean) => {
+const generateFields = (
+  answersCompany: ApplicationCompany,
+  answersBusiness: ApplicationBusiness,
+  referenceNumber: number,
+  checkAndChange: boolean,
+): Array<SummaryListGroupData> => {
   const fields = [
-    ...generateYourCompanyFields(answersCompany, referenceNumber, checkAndChange),
-    ...generateNatureOfYourBusinessFields(answersBusiness, referenceNumber, checkAndChange),
-    ...generateTurnoverFields(answersBusiness, referenceNumber, checkAndChange),
-    ...generateCreditControlFields(answersBusiness, referenceNumber, checkAndChange),
-  ] as Array<SummaryListItemData>;
+    generateYourCompanyFields(answersCompany, referenceNumber, checkAndChange),
```

Comment:
do we not need `...` here?

Reply:
No, it needs an array of objects now instead of an array of strings. Previously it was just an array of fields for a single summary list, now it's an array of summary lists.